packages/confluence-adapter/src/error_utils.ts

```diff
@@ -18,26 +18,39 @@ import _ from 'lodash'
 import { isSaltoElementError, isSaltoError } from '@salto-io/adapter-api'
 import { deployment } from '@salto-io/adapter-components'
 
+const isJavaNullPointerErrorOrUndefined = (err: Error): boolean => {
+  const status = _.get(err, 'response.status')
+  const message = _.get(err, 'response.data.message')
+  return status === 500 && message.startsWith('java.lang.NullPointerException: Cannot invoke')
+}
+
+const suppressedErrorsCheckerList: ((err: Error) => boolean)[] = [isJavaNullPointerErrorOrUndefined]
```

Comment:
nits:
1. maybe define a type for the error checker signature?
2. this is a constant so should be 
```suggestion
const SUPPRESSED_ERRORS_CHECKER_LIST: ((err: Error) => boolean)[] = [isJavaNullPointerErrorOrUndefined]
```